contracts/pcv/utils/PCVDripController.sol

```diff
@@ -68,7 +68,9 @@ contract PCVDripController is
         _initTimed();
 
         // incentivize caller
-        _incentivize();
+        if (incentiveAmount != 0) {
+            _incentivize();
+        }
```

Comment:
this is redundant. _incentivize calls _mintFei which also does a 0 check

Reply:
removed